src/vars.rs

```diff
@@ -30,12 +30,25 @@ pub struct Vars {
     // multiple allocations and lifetime tracking.
     blank: Var,
     default_recipe_prefix: Var,
+    make_var: Var,
+    makeargs_var: Var,
 }
 
 impl Vars {
     /// Primary interface for configuring a new instance. We also create some cached values that
     /// live for the lifetime of this instance to reduce the number of allocations.
     pub fn new<const N: usize>(init: [(&str, &str); N]) -> Self {
+        // Get the executable path for the `MAKE` variable.
+        let exe_path = match std::env::current_exe() {
+            // TODO: This should probably be a `Result` instead of a `panic!`.
+            // Try to canonicalize the path, but if that fails we panic because
+            // this means something is very wrong with the system.
+            Ok(path) => path.canonicalize().unwrap().to_string_lossy().into_owned(),
+            Err(_) => panic!("Unable to get executable path!"),
+        };
+
+        
+
```

Comment:
I think a single blank line between relevant code blocks is sufficient, not 3.

Reply:
formatting misshap, I agree lol